webpack/stories/docs/APIMiddleware.md

```diff
@@ -46,3 +46,27 @@ dispatch({
   }
 });
 ```
+
+## Main API Reducer
+
+The API will manage the request lifecycle in the redux store > API > the provided key, e.g:
+```js
+{
+  API: {
+    MY_COMPONENT_KEY: { // this is the key you provided in the API action.
+      status: 'RESOLVED',
+      error: null,
+      ...payload, // the payload we passed to the API action
+      ...data, // the data that was returned in the API response
```

Comment:
It feels risky to me, I would much prefer it as `{ payload, data }`

Reply:
this is the way we also handle it at the moment..
blocked by #7301